SpatialGDK/Source/SpatialGDKTests/SpatialGDKServices/LocalDeploymentManager/LocalDeploymentManagerUtilities.cpp

```diff
@@ -10,20 +10,23 @@
 
 #include "CoreMinimal.h"
 
-namespace
+namespace SpatialGDK
 {
-// TODO: UNR-1969 - Prepare LocalDeployment in CI pipeline
-const double MAX_WAIT_TIME_FOR_LOCAL_DEPLOYMENT_OPERATION = 30.0;
-
-const FName AutomationWorkerType = TEXT("AutomationWorker");
-const FString AutomationLaunchConfig = FString(TEXT("Improbable/")) + *AutomationWorkerType.ToString() + FString(TEXT(".json"));
-
 FLocalDeploymentManager* GetLocalDeploymentManager()
 {
 	FSpatialGDKServicesModule& GDKServices = FModuleManager::GetModuleChecked<FSpatialGDKServicesModule>("SpatialGDKServices");
 	FLocalDeploymentManager* LocalDeploymentManager = GDKServices.GetLocalDeploymentManager();
 	return LocalDeploymentManager;
 }
+} // namespace SpatialGDK
+
+namespace
+{
+// TODO: UNR-1969 - Prepare LocalDeployment in CI pipeline
```

Comment:
Is this still relevant with standalone?

Reply:
The ticket is actually closed, maybe worth removing?

Reply:
Will remove comment, good catch!